index.html

```diff
@@ -9,6 +9,20 @@
 	<link rel="stylesheet" type="text/css" href="index.css">
 </head>
 <body>
+<div class="wrapper">
+	 
+	 <header class="header">
+	 	<img class="header__logo" src="images/logo.svg" alt="header logo">
```

Comment:
Please get familiar with the [different ways of using SVG](https://css-tricks.com/using-svg/).